### PR TITLE
migrate_mem: add case for large mem

### PR DESCRIPTION
**Pull request by dzhengfy**
The scenario is to test large memory vm's migration.
RHEL7-17399

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
(1/3) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_device: PASS (138.37 s)
 (2/3) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_balloon: PASS (142.32 s)
 (3/3) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.large_mem: PASS (149.52 s)

**Comment by dzhengfy** [comment]
(1/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_device: STARTED
 (1/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_device: PASS (137.01 s)
 (2/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_balloon: STARTED
 (2/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_balloon: PASS (139.57 s)
 (3/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_nvdimm: STARTED
 (3/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_nvdimm: PASS (182.36 s)
 (4/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.large_mem_vm: STARTED
 (4/4) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.large_mem_vm: PASS (145.04 s)
libvirt-8.0.0-8.el9_0.x86_64

File: libvirt/tests/src/migration/migrate_mem.py [hunk 1]
```diff
@@ -403,8 +434,7 @@ def run_migration(vm, params, test):
     migration_test.check_result(mig_result, params)
```
Comment (Yingshun) [inline libvirt/tests/src/migration/migrate_mem.py:434]: As you're touching this file, would you please also help remove migration_test.check_result(mig_result, params)? It's been covered by do_migration. Thanks!

Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_mem.py:434]: Sure, I just noticed this duplication. Removed.

File: libvirt/tests/src/migration/migrate_mem.py [hunk 2]
```diff
@@ -403,8 +434,7 @@ def run_migration(vm, params, test):
...
 
     if int(mig_result.exit_status) == 0:
-        migration_test.ping_vm(vm, params, uri=dest_uri)
-
+        migration_test.post_migration_check([vm], params, uri=dest_uri)
     if check_str_local_log:
         libvirt.check_logfile(check_str_local_log, params.get("libvirtd_debug_file"))
```
[the remark this answers is not in the document]
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_mem.py]: updated

**Comment by dzhengfy** [comment]
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.large_mem_vm: PASS (145.94 s)

**Review by Yingshun — APPROVED** [review]
LGTM

**Comment by dzhengfy** [comment]
@cliping , please help review. Thanks

**Review by cliping — APPROVED** [review]
LGTM

**Pull request merged**